commands/init.go

```diff
@@ -151,3 +157,20 @@ func InitWebSocketClient() (*model.WebSocketClient, error) {
 	}
 	return client, nil
 }
+
+func InitUnixClient() (*model.Client4, error) {
+	httpClient := &http.Client{
+		Transport: &http.Transport{
+			Dial: func(network, addr string) (net.Conn, error) {
+				return net.Dial("unix", viper.GetString("local-socket-path"))
+			},
+		},
+	}
+
+	client := &model.Client4{
+		ApiUrl:     "http://_" + model.API_URL_SUFFIX,
+		HttpClient: httpClient,
+		HttpHeader: map[string]string{},
+	}
```

Comment:
0/5, this could also be re-written as:

```go
client := model.NewAPIV4Client("http://_" + model.API_URL_SUFFIX)
client.HttpClient = &http.Client{Transport: tr}
```

And just initialize the transport above.